src/script/view_model/CallingViewModel.test.ts

```diff
@@ -124,24 +112,14 @@ describe('CallingViewModel', () => {
 
       expect(mockCallingRepository.startCall).toHaveBeenCalledWith(mlsConversation, CALL_TYPE.NORMAL);
 
-      expect(mockCallingRepository.setEpochInfo).toHaveBeenCalledWith(
+      expect(container.resolve(Core).service?.subconversation.subscribeToEpochUpdates).toHaveBeenCalledWith(
```

Comment:
suggestion: We usually do not do this depending pulling in the test. We would usually build the class we are testing with mock dependencies and use those mocked dependencies in the tests